marshmallow/utils.py

```diff
@@ -309,6 +309,11 @@ def from_iso_date(datestring, use_dateutil=True):
     else:
         return datetime.datetime.strptime(datestring[:10], '%Y-%m-%d').date()
 
+
+def to_iso_date(date):
+    return date.strftime('%Y-%m-%d')
```

Comment:
This function already exists in the stdlib as `isoformat` - so we should be able to just use that instead

```python
In [1]: import datetime

In [2]: datetime.date.today().isoformat()
Out[2]: '2018-07-12'
```

Reply:
I'm fine with that. I'll make the change.